addons/website_sale/views/templates.xml

```diff
@@ -468,7 +468,7 @@
         </xpath>
     </template>
 
-    <template id="products_attributes" inherit_id="website_sale.products" active="False" customize_show="True" name="Product Attribute's Filters">
+    <template id="products_attributes" inherit_id="website_sale.products" active="False" customize_show="True" name="Product Attribute's Filters" priority="11">
```

Comment:
Never increment by one (unless there is a reason that prevent you to do otherwise?), or we can't something in between